docs/reference/migration/migrate_7_4.asciidoc

```diff
@@ -133,3 +133,65 @@ deprecated, and will be removed in version 8.0.0. Instead, use `node.pidfile`.
 To ensure that all settings are in a proper namespace, the `processors` setting
 is deprecated, and will be removed in version 8.0.0. Instead, use
 `node.processors`.
+
+[discrete]
+[[breaking_74_transform_changes]]
+=== {transform-cap} changes
+
+[discrete]
+[[transform_stats_format]]
+==== Stats response format changes
+
+The response format of the <<get-data-frame-transform-stats>> is very different
```

Comment:
The `data frame transform` URLs have been updated to `transform` URLs. Related PR: #46940 

```suggestion
The response format of the <<get-transform-stats>> is very different
```